src/main/resources/db/migration/changes/UCSBDates.json

```diff
@@ -27,7 +27,7 @@
                       "autoIncrement": true,
                       "constraints": {
                         "primaryKey": true,
-                        "primaryKeyName": "CONSTRAINT_5"
```

Comment:
Why the change from CONSTRAINT_5 to CONSTRAINT_7?

Reply:
it said that CONSTRAINT_5 was taken, so I switched to a different key name